src/pool/runner.rs

```diff
@@ -84,3 +63,14 @@ pub trait RunnerBuilder {
     /// Builds a runner.
     fn build(&mut self) -> Self::Runner;
 }
+
+/// A builder that create new Runner by cloning the old one.
+pub struct CloneRunnerBuilder<R>(pub R);
```

Comment:
What's the purpose of this builder?

Reply:
Provides a convenient `Builder` so that users don't have to write their own. Cloning is a very common use case especially for tests.